templates/alertmanager.go

```diff
@@ -16,13 +16,15 @@ limitations under the License.
 
 package templates
 
+import (
+	promv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
+)
+
 // AlertmanagerTemplate is the template that serves as the base for the Alert Manager deployed by the operator
-const AlertmanagerTemplate = `
-apiVersion: monitoring.coreos.com/v1
-kind: Alertmanager
-spec:
-  replicas: 3
-  alertmanagerConfigSelector:
-    matchLabels:
-      app: managed-ocs
-`
+var _3 = int32(3)
+
+var AlertmanagerTemplate = promv1.Alertmanager{
+	Spec: promv1.AlertmanagerSpec{
+		Replicas: &_3,
+	},
+}
```

Comment:
Considering the changes made in alertmanager config secret name
```suggestion
var AlertmanagerTemplate = promv1.Alertmanager{
	Spec: promv1.AlertmanagerSpec{
		Replicas:     &_3,
		ConfigSecret: "managed-ocs-alertmanager-config-secret",
	},
}

```